sample-build-scripts/react-native/fix-cocoapods-and-node-build/appcenter-post-clone.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
+# This post clone script prepares the build process
+# in App Center for React Native 0.60 projects by
+# resolving a few build errors.
+#
+# Put this script in your ios/ folder and commit.
+#
+# Please mind the version numbers so that they
+# correspond to your Podfile and package.json.
+
+echo "Uninstalling all cocoapods versions"
+sudo gem uninstall cocoapods --all
+echo "Installing cocoapods version 1.7.5"
+sudo gem install cocoapods -v 1.7.5
+
+# Upgrade Node to a version expected by React Native 0.60
+set -ex
+brew uninstall node@6
+NODE_VERSION="8.10.0"
```

Comment:
You can specify Node.js version through UI. If it doesn't work for you, could you please share your use-case?

<img width="620" alt="Screen Shot 2019-09-06 at 2 29 53 PM" src="https://user-images.githubusercontent.com/16715858/64425025-5678ee00-d0b3-11e9-90a8-bda74cc9d2bf.png">

Reply:
Strange, I don't see that option?

![image](https://user-images.githubusercontent.com/30017/64426733-aeadf100-d0af-11e9-9664-367e88602bac.png)

Reply:
@jorisw When you created the app in App Center, did you choose React Native as the platform? If you go to your app settings page now, what does it say as Platform?

Reply:
Ah, my client created a regular iOS Swift / ObjC project.